example_client_test.go

```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
 	// Ranges: [{time-serie-1 map[] [{4 2}]} {time-serie-2 map[] [{4 10}]}]
 	// Ranges with labels: [{time-serie-1 map[az:us-east-1 machine:machine-1] [{4 2}]} {time-serie-2 map[az:us-east-1 machine:machine-2] [{4 10}]}]
 }
+
+// Exemplifies the usage of MultiAdd.
+func ExampleClient_MultiAdd() {
+	host := "localhost:6379"
+	password := ""
+	pool := &redis.Pool{Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", host, redis.DialPassword(password))
+	}}
+	client := redistimeseries.NewClientFromPool(pool, "ts-client-1")
+
+	labels1 := map[string]string{
+		"machine": "machine-1",
+		"az":      "us-east-1",
+	}
+	labels2 := map[string]string{
+		"machine": "machine-2",
+		"az":      "us-east-1",
+	}
+
+	err := client.CreateKeyWithOptions("timeserie-1", redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels1})
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = client.CreateKeyWithOptions("timeserie-2", redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels2})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Adding multiple datapoints to multiple series
+	datapoints := []redistimeseries.Sample{{"timeserie-1", redistimeseries.DataPoint{1, 10.5}}, {"timeserie-1", redistimeseries.DataPoint{2, 40.5}}, {"timeserie-2", redistimeseries.DataPoint{1, 60.5}}}
```

Comment:
@filipecosta90 can you format it a bit better? perhaps each measurement should start a new line.

Reply:
agreed and solved in the latest commit :+1: